code-challenges.js

```diff
@@ -40,6 +82,17 @@ var testString1 = "albatross"
 var testString2 = "jabberwocky"
 // Expected output: "jAbBeRwOcKy"
 
+function firstLetterUppercase (input) {
+  var res = "";
+  for (i=0; i < input.length; i++) {
+     res += i % 2 == 0 ? input.charAt(i).toUpperCase() : input.charAt(i);
+  }
+  return res;
+}
+
+console.log(firstLetterUppercase(testString1));
+console.log(firstLetterUppercase(testString2));
+
```

Comment:
I am curious about your syntax, it looks a little different than the rest of your code. Also, make sure you are paying attention to the details. This code uppercases the letters at even indexes when the expected output is capitalization at odd indexes.